README.md

```diff
@@ -136,10 +136,10 @@ DREX also comes with a simple project drawer functionality
 
 ## Configuration
 
-There is no initial setup needed to use DREX  
+There is no initial setup needed to use DREX
 However you may configure certain settings to your liking
 
-Check out `:help drex-configuration` for more details about the individual options  
```

Comment:
These trailing white spaces are there on purpose to force a line break in markdown